service/chartconfig/v1/helm/helm.go

```diff
@@ -122,7 +122,11 @@ func (c *Client) InstallFromTarball(path, ns string, options ...helmclient.Insta
 
 // UpdateReleaseFromTarball updates the given release using the chart packaged
 // in the tarball.
-// TODO
 func (c *Client) UpdateReleaseFromTarball(releaseName, path string, options ...helmclient.UpdateOption) error {
+	_, err := c.helmClient.UpdateRelease(releaseName, path, options...)
+	if err != nil {
```

Comment:
This is maybe a matter of taste, but you can drop `if` here and just `return microerror.Mask(err)` - if `err == nil`, `Mask()` will return nil as well.

Reply:
I want to keep the explicit way because of the "tracing" we get from the error stacks. They tell you a story. When you take the shortcut here and shit hits the fan the story you get told can be confusing because you might or might now have a mental model of the code already and then the stack says something contradictory to what you may have in mind.